libs/designsystem/checkbox/src/checkbox.component.ts

```diff
@@ -51,9 +60,49 @@ export class CheckboxComponent {
 
   onChecked(checked: boolean): void {
     this.checked = checked;
+    this._onChange(this.checked);
     this.checkedChange.emit(this.checked);
+    this._onTouched();
   }
 
   // IDs used for a11y labelling
   _labelId = UniqueIdGenerator.scopedTo('kirby-checkbox-label').next();
+
+  //ControleValueAcessor
```

Comment:
```suggestion
  // Initialize empty ControlValueAccessor callback functions
```
Consider giving a bit of context in this comment